mqclient/queue.py

```diff
@@ -289,13 +293,13 @@ async def __aenter__(self) -> "MessageAsyncGeneratorContext":
             "[MessageAsyncGeneratorContext.__aenter__()] entered `with-as` block"
         )
 
-        if self.sub and self.gen:
+        if self._sub and self._gen:
```

Comment:
Should this be an `or` conditional?

From the code, it looks like they should be set together. If this isn't the case (let's say a client sets `_sub` or `_gen` from the outside), would you still want the RuntimeError to be thrown if one of these is set?

Reply:
Right, the assumption is that these are always set together. These were made private because the client shouldn't be messing with this. If they do change these attributes, that's their fault